RUN_PROTOCOL.md

```
@@ -0,0 +1,222 @@
+node scripts/auto-submit.mjs `
+  --kanban-json data/kanban-snapshot-$(Get-Date -Format yyyy-MM-dd).json `
+  --dry-run --report --limit 10
+```
+
+**Eligible states** (cards in any other state are filtered out):
+| State | Why eligible |
+|-------|-------------|
+| `evaluated` | User reviewed/scored the card and wants to proceed to application |
+
+**JSON shape contract** (`exportState()` in dashboard/job-pulse-kanban.html):
+```json
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)